postcss.config.js

```diff
@@ -1,3 +0,0 @@
-module.exports = {
```

Comment:
I removed this because I don't think it was being used for `@ithaka/pharos` package since we use `postcss` directly in a script. But now I am curious if this might be used by one of the other pharos packages?

Reply:
Believe it was used for the Storybook build.

Reply:
ah gotcha @Niznikr . Added it back.